src/features/projects/components/projects/Filters.tsx

```diff
@@ -11,10 +11,10 @@ export default function Filters(): ReactElement {
   const { projects, setFilteredProjects, filtersOpen, setFilterOpen } =
     useProjectProps();
 
-  const [purpose, setPurpose] = React.useState({
+  const purpose = {
     restoration: true,
     conservation: true,
-  });
+  };
```

Comment:
`purpose` does not seem required any more - @prachigarg19 please check and remove if not needed.